endpoint.go

```diff
@@ -533,13 +533,16 @@ func (ep *endpoint) rename(name string) error {
 	n.updateSvcRecord(ep, n.getController().getLocalEps(netWatch), false)
 
 	oldName := ep.name
+        oldAnonymous := ep.anonymous
```

Comment:
looks like you're using spaces here for indentation; other lines use tabs; can you change this to tabs?